src/fullscreennotification.cpp

```diff
@@ -7,7 +8,7 @@ FullScreenNotification::FullScreenNotification(QWidget *parent)
     : QLabel(parent)
     , m_previouslyVisible(false)
 {
-    setText(tr("You are now in full screen mode. Press ESC to quit!"));
+    setText(KiwixApp::instance()->getText("fullscreen-notification"));
```

Comment:
It would be nice to have a function or a macro `gt` to do it.
`gt("fullscreen-notification")` instead of `KiwixApp::instance()->getText("fullscreen-notification")`